docs/notebooks/dispersion/two_fluid_dispersion.ipynb

```diff
@@ -132,7 +133,7 @@
     "    ),\n",
     "    \"wci\": pfp.gyrofrequency(inputs[\"B\"], inputs[\"ion\"]),\n",
     "}\n",
-    "params[\"lpe\"] = pfp.inertial_length(params[\"n_e\"], \"e-\")\n",
+    "params[\"lpe\"] = inertial_length(params[\"n_e\"], \"e-\")\n",
     "params[\"wpe\"] = pfp.plasma_frequency(params[\"n_e\"], \"e-\")"
```

Comment:
It's a little inconsistent here in that there's a namespace prefix for `plasma_frequency` but not `inertial_length`.  But, if this is going to be changed when `frequencies.py` gets extracted from `parameters.py`, then there's no need to change that in this PR.

Reply:
Yes, as I do the decomposition I plan to update this notebook with direct imports.